unit_tests/test_scylla_yaml_builders.py

```diff
@@ -134,7 +134,7 @@ def test_aws_multi_openldap(self):
                 'ldap_url_template': 'ldap://1.1.1.1:389/dc=scylla-qa,dc=com?cn?sub?'
                                      '(uniqueMember=uid={USER},ou=Person,dc=scylla-qa,dc=com)',
                 'role_manager': 'com.scylladb.auth.LDAPRoleManager',
-                'saslauthd_socket_path': '/run/saslauthd/mux'
+                'saslauthd_socket_path': '/run/saslauthd/mux',
```

Comment:
not related change, not needed in that case

Reply:
removed